api/v1alpha1/lvmvolumegroup_types.go

```diff
@@ -25,6 +25,9 @@ import (
 
 // LVMVolumeGroupSpec defines the desired state of LVMVolumeGroup
 type LVMVolumeGroupSpec struct {
+	// DeviceClassName is the name of the device class associated with this volume group
+	DeviceClassName string `json:"deviceClassName,omitempty"`
```

Comment:
I'm not convinced this is required.

Reply:
removed. Only supporting one pool per device class/ Volume group now.